DCOS/utils/kill-mesos-agent-ungraceful.ps1

```diff
@@ -0,0 +1,48 @@
+$mesosServiceObj = Stop-Service dcos-mesos-slave -PassThru
+$mesosServiceObj.WaitForStatus('Stopped','00:00:30')
+if ($mesosServiceObj.Status -ne 'Stopped') {
+    Write-Output "Failed to stop the service"
+    exit 1
+} else {
+    $check = sc.exe qc dcos-mesos-slave | Select-String "--recovery_timeout=1mins"
```

Comment:
Check the `$LASTEXITCODE` to do the error checking for the `sc.exe` binary call.